src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,36 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int compareCallCheck = -1;
```

Comment:
W klasie DefaultCountingOutRhymer stałe o wartości -1 odnoszące się do pola total (też w metodzie callCheck), to nie jest ta sama stała o wartości -1 oznaczające domyślną zwracaną przez wyliczankę wartość (w metodach peekaboo i countOut). Powinny to być osobne stałe.